src/mjolnir/admin.cc

```diff
@@ -3,10 +3,11 @@
 #include "filesystem.h"
 #include "midgard/logging.h"
 #include "mjolnir/util.h"
-#include <spatialite.h>
 #include <sqlite3.h>
 #include <unordered_map>
 
+#include <spatialite.h>
```

Comment:
need to put those at the end as sqlite3.h needs to be included before spatialite.h. otherwise format.sh will simply switch the includes

Reply:
also I remove all references to sqlite/spatialite from mjolnir/util.h so we now get a lot more includes in cc files